README.md

```diff
@@ -84,12 +84,16 @@ The content type of the response depends on the request's `Accepts` header.
 | safeFields | [String] | `[]` |  Specifies property names on errors that are allowed to be passed through in 4xx and 5xx responses. See [Safe error fields](#safe-error-fields) below. |
 | defaultType | String | `"json"` | Specify the default response content type to use when the client does not provide any Accepts header.
 | negotiateContentType | Boolean | true | Negotiate the response content type via Accepts request header. When disabled, strong-error-handler will always use the default content type when producing responses. Disabling content type negotiation is useful if you want to see JSON-formatted error responses in browsers, because browsers usually prefer HTML and XML over other content types.
+| htmlRenderer | String | 'send-html' | Operation function to render HTML with signature `fn(res, data)`. Defaults to `./send-html` (`lib/send-html.js`). |
```

Comment:
I like the idea of customizable renderer functions 👍 

I am concerned about the design you are proposing though. Passing in a string path may seem like a convenient solution in the context of LoopBack's `middleware.json`, but there are drawbacks too:

1. When using strong-error-handler in vanilla Express applications, configuring it directly from JavaScript code, the usual convention is to pass functions, not paths to source code files. 

    ```js
    const app = express();
    app.use(strongErrorHandler({
      log: true,
      debug: false,
      htmlRenderer: function(res, data) {
        // my code
      },
    });
    ```

    This can be addressed by modifying the implementation to recognize both String and Function values.

2. Using a path that's directly passed to `require` can lead to situation difficult to debug. Consider what happens when you forget to include `$!` prefix in your `middleware.json` entry: the path will be resolved relatively  to strong-error-handler. Either a wrong file will be loaded, or what's more likely, "module not found" error will be printed.

    I am proposing to address this usability issue by checking whether the user-supplied path is absolute, and throwing an error with a helpful error message otherwise.

Thoughts?

Also please change the description of the default value. Consumers of this module should not need to understand our internal implementation. IMO, it's better to tell them that by default the built-in renderer is used. In fact, I think the rendered options should not have any default value per se, instead we should interpret a missing value as "use the built-in renderer".

```
| htmlRenderer | String | | An optional function to render HTML, the function should have signature `fn(res, data)`. Use this option to override the built-in HTML renderer.
```

Reply:
1. Absolutely agree, would prefer it be definable as a function as well. 

2. If a path is supplied, are you suggesting it would be required to start with `$!` or else we'd throw a helpful error? I was thinking it would behave however core LoopBack behaves in this regard, where I thought you could do it with or without the `$!`.

3. Good point.

Reply:
> If a path is supplied, are you suggesting it would be required to start with `$!` or else we'd throw a helpful error? I was thinking it would behave however core LoopBack behaves in this regard, where I thought you could do it with or without the `$!`.

The prefix `$!` is something that loopback-boot recognizes and expands into an absolute path. LoopBack runtime itself (loopback, strong-remoting, express) never see that value!

 - Here in strong-error-handler, we should enforce absolute paths to prevent user confusion.
 - The documentation for LoopBack users should use `$!` in the examples, that's how this new feature should be used in LoopBack apps
 - loopback-boot will take care of the rest

Thoughts?